sdk/quantum/Azure.Quantum.Jobs/api/Azure.Quantum.Jobs.netstandard2.0.cs

```diff
@@ -0,0 +1,32 @@
+namespace Azure.Quantum.Jobs
```

Comment:
You need to re-export your final APIs per https://github.com/Azure/azure-sdk-for-net/blob/master/CONTRIBUTING.md#public-api-additions or the CI will fail.

Reply:
@george-moussa